tests/test_cli.py

```diff
@@ -13,15 +13,29 @@
 KEEP_GENERATED = os.environ.get("KEEP_GENERATED", False)
 UPDATE_EXPECTED = os.environ.get("UPDATE_EXPECTED", False)
 COMPILERS = {
-    # cpp is disabled due to https://github.com/adsharma/py2many/issues/24
-    # "cpp": ["clang", "-std=c++14"],
-    "dart": ["dart"],
+    "cpp": ["clang", "-std=c++14"],
+    "dart": ["dart", "compile", "exe"],
     "go": ["go", "tool", "compile"],
-    "julia": ["julia"],
-    "kotlin": ["kotlinc", "-p", "library"],
```

Comment:
"-p library" allows kotlin to be compiled without main()

Reply:
`kotlinc-jvm` doesnt support that.